symfony/framework-bundle/7.2/config/packages/framework.yaml

```diff
@@ -2,6 +2,10 @@
 framework:
     secret: '%env(APP_SECRET)%'
 
+    form: { csrf_protection: { token_id: submit } }
```

Comment:
This makes every form use the same token ID from what I understand. But this is in constrast to the recommendation at https://symfony.com/doc/current/security/csrf.html#csrf-protection-in-symfony-forms

> // an arbitrary string used to generate the value of the token
            // using a different string for each form improves its security
            'csrf_token_id'   => 'task_item',

If that is not true, it should be changed. What's the point of the token ID then at all? Just a way to distinguish different forms?

Reply:
Thanks for the comments. I agree docs on the topic need to be changed after this PR.
(Yes, token id is to differentiate by use case for each tokens, which means by forms most of the time, but this is not required for the CSRF protection to be effective)